### ISSUE TITLE
"selector-newline" false warning

### ISSUE THREAD

**Issue by XhmikosR** [CSSLint/csslint#490]
Originally from [here](https://github.com/CSSLint/csslint/commit/086b153adb2e3848f1b658604582358a113d2fc5#commitcomment-5965907).

@mahonnaise @nschonni 

This seems to yield wrong results for `>`, `+` etc selector.

``` css
.latest-news > h1 {
    padding-left: 36px;
}
```
